io.c

```diff
@@ -15160,39 +15168,52 @@ set_LAST_READ_LINE(VALUE val, ID _x, VALUE *_y)
  *    f.gets        # => "e\n"
  *    f.close
  *
- *  There are two special line separators:
+ *  Or by setting global variable <tt>$/</tt>:
+ *
+ *    f = File.new('t.txt')
+ *    $/ = 'l'
+ *    p f.gets
+ *    p f.gets
+ *    p f.gets
```

Comment:
I think this is more consistent with the rest of the docs:

```suggestion
 *    f.gets # => "First l"
 *    f.gets # => "ine\nSecond l"
 *    f.gets # => "ine\n\nFourth l"
```

Reply:
FIxed (and "Output" below removed).